ui/src/app/pages/admin/user-audit.tsx

```diff
@@ -0,0 +1,46 @@
+import {profileApi} from 'app/services/swagger-fetch-clients';
+import * as fp from 'lodash/fp';
+import * as React from 'react';
+import {useParams} from 'react-router-dom';
+import {AuditAction, AuditEventBundle} from '../../../generated';
+import {AuditPageComponent} from '../../components/admin/audit-page-component';
+
+const getAuditLog = (subject: string) => {
+  const bqRowLimit = 1000; // Workspaces take many rows because of the Research Purpose fields
+  return profileApi().getAuditLogEntries(subject, bqRowLimit);
+};
+
+const queryAuditLog = (subject: string) => {
+  return getAuditLog(subject).then((queryResult) => {
+    console.log(queryResult.query);
```

Comment:
is this necessary?

Reply:
I like it, because it gives you a jumping-off point in BigQuery for follow-up exploration (not to mention debugging). It would be nice to show it (optionally) in the UI.